core/game.py

```diff
@@ -136,24 +129,33 @@ def add_answer(self, filled_poll: FilledPoll):
         ] = filled_poll.answers
         return True
 
-    def get_remaining_poll_targets(self, user) -> List[User]:
+    def get_remaining_poll_targets(self, user: User) -> List[str]:
         """
         :param user: user for whom it is checked for which users answers are to be filled
         :return: list of user for whom polls are to be filled
         """
         return [
-            person
+            person.name
             for person in self.user_data[user]
             if self.user_data[user][person] is None
         ]
 
+    def get_answers_about(self, user: User) -> Dict[str, List[dict]]:
+        return {u.name: [asdict(i) for i in self.user_data[u][user]]
+                for u in self.user_data if (u != user and self.user_data[u][user] is not None)}
+
```

Comment:
Generalnie to powinno zwracać obiekty, a JSONa robimy w routerze. Już przerobione, ale do sprawdzenia

Reply:
ok

Reply:
Czyli mam rozumieć, że słownik obiektów zamiast słownika stringów?